pkg/execute/executor.go

```diff
@@ -146,8 +144,21 @@ func runKubectlCommand(args []string, clusterName string, isAuthChannel bool) st
 		}
 		if strings.HasPrefix(arg, ClusterFlag.String()) {
 			if arg == ClusterFlag.String() {
+				if index == len(args)-1 {
+					return ""
+				}
+				arg = args[index+1]
```

Comment:
@mugdha-adhav can we use `continue` here?

Reply:
As discussed, will add comments to make code readable.